src/deps/brotli_decoder.zig

```diff
@@ -152,6 +154,38 @@ pub const BrotliDecoderErrorCode = enum(c_int) {
     ALLOC_BLOCK_TYPE_TREES = -30,
     UNREACHABLE = -31,
 };
+pub const BrotliDecoderErrorCode2 = enum(c_int) {
```

Comment:
Why is it called `BrotliDecoderErrorCode2`?

Reply:
uses different field names, to cover the possible codes more exhaustively. didn't want to break existing code in this branch